mmv1/products/accesscontextmanager/ServicePerimeter.yaml

```
@@ -54,26 +54,22 @@ mutex: '{{name}}'
 examples:
   - !ruby/object:Provider::Terraform::Examples
     name: 'access_context_manager_service_perimeter_basic'
-    skip_test: true
     primary_resource_id: 'service-perimeter'
     vars:
       access_level_name: 'chromeos_no_lock'
       service_perimeter_name: 'restrict_storage'
   - !ruby/object:Provider::Terraform::Examples
     name: 'access_context_manager_service_perimeter_secure_data_exchange'
-    skip_test: true
     primary_resource_id: 'secure-data-exchange'
     vars:
       access_level_name: 'secure_data_exchange'
   - !ruby/object:Provider::Terraform::Examples
     name: 'access_context_manager_service_perimeter_dry-run'
```

```suggestion
    name: 'access_context_manager_service_perimeter_dry_run'
```

Please rename the example file that this line is referencing to also have `_` instead of `-`